Makefile

```diff
@@ -26,96 +27,112 @@ unittest-dev-222:
 	@sudo rm -rf /home/ansys/.local/share/ansys_fluent_core/examples
 	@pip install -r requirements/requirements_tests.txt
 	@python -m pytest --fluent-version=22.2 $(PYTESTEXTRA)
+	@python -m pytest --fluent-version=22.2 $(PYTESTRERUN) $(PYTESTEXTRA)
```

Comment:
@prmukherj Thinking about this again, the re-run should not include the coverage analysis as all tests won't be run. I think the github failure (Linux error code 5) is due to the covergae file is not found.